UKGovernmentBEIS/beis-primary-authority-register
# PAR-1172: Make organisation de-duping screens ignore coordinated member organisations

To test:
Use any coordinated business entity, find its coordinated members and try search for them, it should show no results.
Use a direct partnership organisation to validate that these correctly show.

How it works:
Checks if the organisation that the suggestion form is going to display is in any of the coordinated_business entities.
